Dockerfile-8.x

```diff
@@ -40,6 +40,20 @@ RUN     yum install -y \
             wget \
             which
 
+# Build extras for test
+RUN     yum install -y \
+            clang-analyzer \
+            libasan \
+            valgrind
+
+# Building with Xen and dom0 Linux libs
+RUN     yum install -y \
+            xen-dom0-libs-devel \
+            openssl-devel \
+            libxml2-devel \
+            libxml2-devel-devel \
+            libseccomp-devel
+
```

Comment:
This is not how the build env is intended to be used. It's must remain minimal (and it's already bigger than koji's minimal build root actually) so that only the required dependencies are present during the build.

* When we build a RPM, the dependencies are installed using yum-builddep
* For manual builds, you need to install the deps by yourself or script their installation, but nothing forbids you to create a separate build env container that is dedicated to some builds you do often, or to make a container persistent and reuse it.

Reply:
Gotcha.  I was wondering where that was done.

Well, I've been trucking along building my container so will continue like that.  I'll remove this piece.